### PR TITLE
Adding --cpuprofile option to prepack-cli

### PR DESCRIPTION
**Pull request by NTillmann**
Release notes: None

Adding ability to record CPU profile traces from node/v8.

### CODE REVIEW

File: src/prepack-cli.js [hunk 1]
```diff
@@ -228,6 +230,10 @@ function run(
           reproFilePath = args.shift();
           // skip for repro purposes
           break;
+        case "cpuprofile":
+          cpuprofilePath = args.shift();
+          // skip for repro purposes
```
Comment (hermanventer) [inline src/prepack-cli.js]: This leaves me perplexed rather than enlightened.

Comment (NTillmann) [reply src/prepack-cli.js]: What exactly is perplexing?

Comment (hermanventer) [reply src/prepack-cli.js]: The comment. I now notice it is following an existing pattern. Nevertheless, what on earth is being skipped  and how does skipping help with repro purposes?

Comment (hermanventer) [reply src/prepack-cli.js]: Coming back to me now. Perhaps "do not include this in options produced by --repro".

Comment (NTillmann) [reply src/prepack-cli.js]: "Skipping" it means the repro-command-line that is synthesized is not going to mention that cpuprofile path. My reasoning for not including is that 1) it's really unlikely to affect behavior, and 2) this output path might not be portable across environments.

Comment (NTillmann) [reply src/prepack-cli.js]: Will make comments more enlightening...

**Pull request closed without merging**